File: Java/Tema1_Ejercicio1.java
/*
 * TEMA 1: INTRODUCCIÓN BÁSICA - EJERCICIO 1
 * ==========================================
 * Escriba un programa que lea dos números desde la entrada estándar 
 * y muestre en la salida la suma, resta, multiplicación y división.
 * 
 * EXPLICACIÓN:
 * - Usamos la clase Scanner para leer entrada del usuario
 * - Realizamos operaciones aritméticas básicas
 * - Mostramos los resultados con println()
 */

import java.util.Scanner;

public class Tema1_Ejercicio1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Lectura de dos números
        System.out.print("Digite un número: ");
        int numero1 = scanner.nextInt();
        
        System.out.print("Digite otro número: ");
        int numero2 = scanner.nextInt();
        
        // Operaciones aritméticas
        int suma = numero1 + numero2;
        int resta = numero1 - numero2;
        int multiplicacion = numero1 * numero2;
        double division = (double) numero1 / numero2;
        
        // Mostrar resultados
        System.out.println("\n=== RESULTADOS ===");
        System.out.println("Suma: " + suma);
        System.out.println("Resta: " + resta);
        System.out.println("Multiplicación: " + multiplicacion);
        System.out.println("División: " + division);
        
        scanner.close();
    }
}

/*
 * SALIDA ESPERADA (Ejemplo con 10 y 3):
 * ====================================
 * Digite un número: 10
 * Digite otro número: 3
 * 
 * === RESULTADOS ===
 * Suma: 13
 * Resta: 7
 * Multiplicación: 30
 * División: 3.3333333333333335
 */

File: Python/Tema1_Ejercicio1.py
"""
TEMA 1: INTRODUCCIÓN BÁSICA - EJERCICIO 1
==========================================
Operaciones aritméticas básicas.

EXPLICACIÓN:
- Función print() para salida
- Variables y tipos de datos
- Operaciones: suma, resta, multiplicación, división
"""

# Números enteros
num1 = 10
num2 = 3

print("=== OPERACIONES ARITMÉTICAS ===")
print(f"Números: {num1} y {num2}")
print(f"\nSuma: {num1} + {num2} = {num1 + num2}")
print(f"Resta: {num1} - {num2} = {num1 - num2}")
print(f"Multiplicación: {num1} * {num2} = {num1 * num2}")
print(f"División: {num1} / {num2} = {num1 / num2:.2f}")
print(f"División entera: {num1} // {num2} = {num1 // num2}")
print(f"Residuo: {num1} % {num2} = {num1 % num2}")
print(f"Potencia: {num1} ** {num2} = {num1 ** num2}")

# Variables flotantes
pi = 3.14159
radio = 5

area = pi * radio ** 2
print(f"\n=== CÁLCULO DE ÁREA ===")
print(f"Área de círculo con radio {radio}: {area:.2f}")

"""
SALIDA ESPERADA:
=== OPERACIONES ARITMÉTICAS ===
Números: 10 y 3

Suma: 10 + 3 = 13
Resta: 10 - 3 = 7
Multiplicación: 10 * 3 = 30
División: 10 / 3 = 3.33
División entera: 10 // 3 = 3
Residuo: 10 % 3 = 1
Potencia: 10 ** 3 = 1000

=== CÁLCULO DE ÁREA ===
Área de círculo con radio 5: 78.54
"""
